apis/python/test/test_index.py

```diff
@@ -344,6 +347,14 @@ def test_ivf_pq_index(tmp_path):
     query_and_check_distances(
         index, np.array([[2, 2, 2]], dtype=np.float32), 2, [[0, 3]], [[2, 1]]
     )
+    query_and_check_distances(
+        index,
+        np.array([[2, 2, 2]], dtype=np.float32),
+        2,
+        [[0, 3]],
+        [[2, 1]],
+        k_factor=2.0,
```

Comment:
Let's also add a test without specifying `k_factor` argument.

Reply:
Mmm sorry, don't follow, can you explain more?

Reply:
Ah, we already have that right above this though, so I think we're okay?